softserve/views.py

```diff
@@ -80,35 +72,61 @@ def dashboard():
 
 
 @app.route('/create_node', methods=['GET', 'POST'])
-#@organization_access_required('gluster')
+@organization_access_required('gluster')
 def get_node_data():
-    count = db.session.query(func.count(Vm.id)) \
-            .filter_by(state = 'ACTIVE').scalar()
-    if count >=5:
-        flash('Limit got over!')
-        return render_template('home.html')
+    if request.method == "POST":
+        counts = request.form['counts']
+        name = request.form['node_name']
+        hours_ = request.form['hours']
+        pubkey_ = request.form['pubkey']
+
+        count = db.session.query(func.count(Vm.id)) \
```

Comment:
Why not do this before the if block and you don't have to repeat the code twice.